nixos/modules/services/databases/postgresql.nix

```diff
@@ -161,33 +161,6 @@ in
               '';
             };
 
-            ensurePermissions = mkOption {
```

Comment:
Shouldn't we have a mkRemovedOption?

Reply:
@SuperSandro2000 do you know how to reasonably implement this for submodules? I'm not aware of any way sadly.

Reply:
We can't just place mkRemovedOptionModule in the submodule? 😅

Reply:
It generates a `config.assertions` block next to the `options.option_that_is_removed` and this won't work in a submodule.